docs/reference/modules/admin-bar.md

```diff
@@ -0,0 +1,125 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/admin-bar`
+
+**Alias:** `apos.adminBar`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module implements Apostrophe's admin bar at the top of the page. Any module can register a button (or more than one) for the bar by calling this module's `add` method. Buttons can also be grouped into dropdown menus and restricted to those with particular permissions. The [apostrophe-pieces](/reference/modules/piece-type.md) takes advantage of this module automatically.
+
+## Configuration options
+
+Options are passed into the admin-bar module by "extending" it through an `index.js` file typically located in the project modules folder, nested inside an `admin-bar` subfolder, inside a `@apostrophecms` folder.
+For example
+```
+deployment
+lib
+modules
+┗ @apostrophecms
+┆              ┗ admin-bar
+┆                        ┗ i18n
+┆                        ┆    ┗ en.json
+┆                        ┆      fr.json
+┆                          index.js 👈
+@node_modules
+app.js
+```
+
+The options passed in this manner will configure the existing `node_modules->apostrophe/lib/modules/admin-bar` module options through implicit subclassing. This same type of configuration takes place when you create a `modules->@apostrophecms->pages->index.js` file in the project to add configuration to the main `page` module.
+
+|  Property | Type | Description |
+|---|---|---|
+|`groups` | Array | Adds one or more menu item group objects to be displayed in dropdown menus |
+
+### `groups`
+
+The `groups` option takes an array of one or more objects that group several menu items together in the admin bar as a dropdown menu. Each of the `groups` objects requires a `label` and an array of menu `items`. The `label` will be used as the label displayed in the menu. The `items` array contains the names of the individual menu items you want to appear in the dropdown, entered in the order you want them to appear. Note: Menu names for `piece-type` items are the name of the piece-type, not the label. For core items, like 'Images', the name is prefixed - `@apostrophecms/image`.
```

Comment:
The name of the piece-type module in question, not the label.

Reply:
Changed